src/index.css

```diff
@@ -1,8 +1,9 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+  /* font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+    sans-serif; */
+    font-family: 'Lato';
```

Comment:
Suggestion: I'd rather add Lato font as the first in list than be the only font. That way, if something goes wrong making the API call to Google Fonts, we still have sensible fallbacks.

Reply:
Added Lato and Monserrat to the front of the font family.